tests/cases/fourslash/superFix2.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path='fourslash.ts' />
+
+////class Base{
+////}
+////class C extends Base{
+////    private a:number;
+////    constructor() {
+////        this.a = 12;

```

Comment:
It would be better if verify.codeFixAtPosition took a range instead. Then you could verify something like
`[|this.a = 12; super()|]` with

`verify.codeFixForRange(range, 'super(); this.a=12;')`